Recrutify/Recrutify.Host/appsettings.json

```diff
@@ -6,5 +6,13 @@
       "Microsoft.Hosting.Lifetime": "Information",
       "Microsoft.AspNetCore.Authorization": "Information"
     }
+  },
+  "MailSettings": {
```

Comment:
you can remove it - you'll add it in Heroku as Mongo, Cors, Authority

Reply:
you have it is in appsettings.Development it's enough. You have to add this data to Heroku anyway, it's unnecessary to add it here with this useless placeholders